### PR TITLE
Feature/550 personality type styles

### PR DESCRIPTION
**Pull request by luxumbra**
Further to @dysbulic initial work, I have added in the updated colours and icons from the designs in Figma and also fixed an issue with the colour bar not filling the full width of the parent container. This PR, when passed, will fix issue #550 in the #501 (MyMeta) roadmap.

### CODE REVIEW

**Review by alalonde — APPROVED** [review]
Nice, I think this looks much better.

**Comment by alalonde** [comment]
Can you reference the originating issue in the description of this PR? And if there's not one already, create it. Will help with QA / tracking

**Review by Baconaetor — APPROVED** [review]
Wow, this looks sooo much better! Still need to do something about the font, and we should change the color for white. I absolutely love the new bar design though, it looks so much nicer and fits into the page well.

**Comment by luxumbra** [comment]
> Can you reference the originating issue in the description of this PR? And if there's not one already, create it. Will help with QA / tracking

Just added that info. :)

**Comment by luxumbra** [comment]
> Wow, this looks sooo much better! Still need to do something about the font, and we should change the color for white. I absolutely love the new bar design though, it looks so much nicer and fits into the page well.

I just noticed that there's a copy pasta mistake and I didn't add the second colour on this gradient. Doing so now. Still not a huge fan of the grey though
image

**Comment by Baconaetor** [comment]
Oooh, that does look much better. I agree the grey is still weird though. I should stay warm and not shift from warm to cool. Does the black gradient include pure black? If so we could through pure white into the white one.

**Comment by dan13ram** [comment]
@luxumbra are you working on this still (re: grey color) or can we merge this ?

**Comment by luxumbra** [comment]
> @luxumbra are you working on this still (re: grey color) or can we merge this ?

I had finished but if it is agreed that grey become white, I'll make a quick update before we merge.

File: packages/web/components/Player/ColorBar.tsx [hunk 1]
```diff
@@ -57,25 +65,34 @@ export const ColorBar = ({
           );
         })}
       </Flex>
-      <Flex minH="calc(1.5rem + 4px)" maxW="100%" border="2px" borderRadius={3}>
+      <Flex
+        minH="calc(1.5rem)"
```
Comment (dysbulic) [inline packages/web/components/Player/ColorBar.tsx:69]:
```suggestion
        minH="1.5rem"
```

File: packages/web/components/Player/ColorBar.tsx [hunk 2]
```diff
@@ -57,25 +65,34 @@ export const ColorBar = ({
...
+            />
           ),
         )}
       </Flex>
-      <FlexContainer mt={1}>
+      <FlexContainer
+        mb={2}
+        fontFamily="mono"
+        sx={{
+          q: { '&::before': { content: "''" }, '&::after': { content: "''" } },
```
Comment (dysbulic) [inline packages/web/components/Player/ColorBar.tsx:93]: It seems like if you're removing the quote characters from the `q` element, you could just use a `Box` instead.

**Review by dysbulic — APPROVED** [review]
I had a couple minor formatting comments, but in general looks great.

**Comment by davort** [comment]
> > @luxumbra are you working on this still (re: grey color) or can we merge this ?
> 
> I had finished but if it is agreed that grey become white, I'll make a quick update before we merge. I thought that Dave @davort may want to chime in from the designers side. :)
> image
> 

Top class job!

Let's go with white, please 😉

**Comment by luxumbra** [comment]
I will just tend to the comments made my @dysbulic and push in 10.

File: packages/web/components/Player/ColorBar.tsx (hunk 2 above)
Comment (luxumbra) [reply packages/web/components/Player/ColorBar.tsx:93]: I have replaced with a `<Box as="span">`

File: packages/web/components/Player/ColorBar.tsx (hunk 1 above)
Comment (luxumbra) [reply packages/web/components/Player/ColorBar.tsx:69]: With `calc()`, this is fine but when i remove it and just have `"1.5rem"` it throws a formatting error. 🤷🏻  
image

image

Comment (dan13ram) [reply packages/web/components/Player/ColorBar.tsx:69]: @luxumbra  you shouldn't have a semi colon there

**Comment by dan13ram** [comment]
> I will just tend to the comments made my @dysbulic and push in 10.

Awesome, please go ahead and merge it yourself once you're done with those changes.
Excited to these changes live ❗

**Comment by luxumbra** [comment]
> > I will just tend to the comments made my @dysbulic and push in 10.
> 
> Awesome, please go ahead and merge it yourself once you're done with those changes.
Thanks fam but looks like I don't have permission to merge.

**Comment by luxumbra** [comment]
Nevermind. 😄 I thought I was a collaborator due to being part of the mages of metagame.  thanks for merging. 🙏🏻

**Pull request merged**